src/org/omegat/core/data/PrepareTMXEntry.java

```diff
@@ -107,5 +147,12 @@ public String toString() {
                 .append(", otherProperties=").append(otherProperties).append("]");
         return builder.toString();
     }
+    
+    ExternalTMXEntry toExternalTMXEntry() {
+        return new ExternalTMXEntry(this);
+    }
 
+    ProjectTMXEntry toProjectTMXEntry(boolean defaultTranslation, ProjectTMXEntry.ExternalLinked linked) {
```

Comment:
Could this change to `public` because  [TMXWriter2 improvement conflicts with this](https://github.com/omegat-org/omegat/pull/185#issuecomment-1017136973)

Reply:
There is another idea to introduce `static ProjectTMX#createPseudoTranslateTMX` in `org.omegat.core.data` package instead of change to `public`.

Reply:
This method is package-protected because previous constructor TMXEntry(PrepareTMXEntry) was like that and the comments in the top of this class said that it was intentional.
Now look to my answer to your previous comment and tell me if it is still required